README.md

```diff
@@ -246,18 +467,46 @@ These are the options available in the `bsconfig.json` file.
             "codes": [1000, 1011] //ignore these specific codes from vendor libraries
         }]
         ```
- - **diagnosticLevel**: `'hint' | 'info' | 'warn' | 'error'` - Specify what diagnostic levels are printed to the console. This has no effect on what diagnostics are reported in the LanguageServer. Defaults to 'warn'
 
- - **autoImportComponentScript**: `bool` - BrighterScript only: will automatically import a script at transpile-time for a component with the same name if it exists.
+Defaults to `undefined`.
+
+#### `diagnosticLevel`
+
+Type: `"hint" | "info" | "warn" | "error"`
```

Comment:
Single quotes changed to double quotes because most of this section is using JSON

Reply:
Sounds good. :)